### PR TITLE
Issue #3064540 by chr.fritsch, daniel.bosen: Remove unneded packages

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-4.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -82,7 +82,6 @@
     "require": {
         "composer/installers": "^1.2",
         "cweagans/composer-patches": "^1.6.5",
-        "marc1706/fast-image-size": "^1.1.0",
```
Comment (dbosen) [inline composer.json:85]: Was this added because of min-testing before?

Comment (chrfritsch) [reply composer.json:85]: Yes, that's how I understand it from the git history.

Comment (dbosen) [reply composer.json:85]: It looks, we do not need it anymore, because of the amp removal

Comment (dbosen) [reply composer.json:85]:
```
composer why marc1706/fast-image-size
thunder/thunder-develop  3.x-dev  requires  marc1706/fast-image-size (^1.1.0)  
lullabot/amp             1.1.4    requires  marc1706/fast-image-size (1.*)
```

Comment (chrfritsch) [reply composer.json:85]: Ah ok, and because lullabot/amp's requirement was to loose, we had to add our own.

Comment (dbosen) [reply composer.json:85]: exactly, so scrap it

Comment (chrfritsch) [reply composer.json:85]: So, approve?

**Review by dbosen — APPROVED** [review]

**Pull request merged**